### ISSUE TITLE
viapplogs.net

### ISSUE THREAD

**Issue by holequantum** [badmojr/1Hosts#597]
i am in pro kindly block this domain `viapplogs.net` this for collecting app usage logs nothing to do with app functionality. it is used in vi app.

**Comment by badmojr** [comment]
Would you please share a link to the app?

**Comment by holequantum** [comment]
> 

https://play.google.com/store/apps/details?id=com.mventus.selfcare.activity

**Comment by badmojr** [comment]
The app does not collect user data according to[ this](https://play.google.com/store/apps/datasafety?id=com.mventus.selfcare.activity).

**Comment by holequantum** [comment]
> The app does not collect user data according to[ this](https://play.google.com/store/apps/datasafety?id=com.mventus.selfcare.activity).

this does not correct apps can say what ever they want this haven't been enforced like ios app tracking protection by which they can verify the claim. this feature is just a cheap copy of apples app transparency and playstores moat apps still don't provide any info. hence it can't be trusted if after block a url app still works then that urls nothi
to do with apps core functionality.
https://reports.exodus-privacy.eu.org/en/reports/269690/

**Comment by holequantum** [comment]
your 1host pro lineup also does not block this domain
`tags.tiqcdn.com`
blocked by 
NextDNS Ads & Trackers Blocklist, Goodbye Ads, notracking, Lightswitch05 - Ads & Tracking , Energized Ultimate.
i think you need to add this domain also.

**Comment by badmojr** [comment]
Cool!
Please provide the exact full domain that's being displayed on your logs.

**Comment by badmojr** [comment]
> your 1host pro lineup also does not block this domain `tags.tiqcdn.com` blocked by NextDNS Ads & Trackers Blocklist, Goodbye Ads, notracking, Lightswitch05 - Ads & Tracking , Energized Ultimate. i think you need to add this domain also.

It would be better to open a new issue instead.

**Comment by holequantum** [comment]
> > your 1host pro lineup also does not block this domain `tags.tiqcdn.com` blocked by NextDNS Ads & Trackers Blocklist, Goodbye Ads, notracking, Lightswitch05 - Ads & Tracking , Energized Ultimate. i think you need to add this domain also.
> 
> It would be better to open a new issue instead.

sure

**Comment by holequantum** [comment]
> Cool! Please provide the exact full domain that's being displayed on your logs.

sure here is it `parse.viapplogs.net`
